packages/@aws-cdk/aws-ecs-patterns/test/ec2/test.l3s-v2.ts

```diff
@@ -0,0 +1,1525 @@
+import { expect, haveResource, haveResourceLike, SynthUtils } from '@aws-cdk/assert';
+import { Certificate } from '@aws-cdk/aws-certificatemanager';
+import ec2 = require('@aws-cdk/aws-ec2');
+import ecs = require('@aws-cdk/aws-ecs');
+import { ApplicationProtocol } from '@aws-cdk/aws-elasticloadbalancingv2';
+import iam = require('@aws-cdk/aws-iam');
+import { PublicHostedZone } from '@aws-cdk/aws-route53';
+import cloudmap = require('@aws-cdk/aws-servicediscovery');
+import cdk = require('@aws-cdk/core');
+import { Test } from 'nodeunit';
+import ecsPatterns = require('../../lib');
+
+export = {
+  'When Application Load Balancer': {
+    'test ECS ALB construct with default settings'(test: Test) {
+      // GIVEN
+      const stack = new cdk.Stack();
+      const vpc = new ec2.Vpc(stack, 'VPC');
+      const cluster = new ecs.Cluster(stack, 'Cluster', { vpc });
+      cluster.addCapacity('DefaultAutoScalingGroup', { instanceType: new ec2.InstanceType('t2.micro') });
+
+      // WHEN
+      new ecsPatterns.ApplicationMultipleTargetGroupsEc2Service(stack, 'Service', {
+        cluster,
+        memoryLimitMiB: 1024,
+        taskImageOptions: {
+          image: ecs.ContainerImage.fromRegistry('test')
+        }
+      });
+
+      // THEN - stack contains a load balancer and a service
```

Comment:
Shouldn't we assert that there are two target groups created as well?

Reply:
I think there's only one target group created with the default setting.